### DISCUSSION TITLE
1 bpp BmpEncoder

### DISCUSSION THREAD

**Discussion by hunkydoryrepair** [SixLabors/ImageSharp#1895] [Ideas] [answered]
I need 1 bpp BMP format for printing. Various thermal printers only accept that format, so it is used a lot for printing logos on receipt, or even UPC codes and QR codes.

**Comment by JimBobSquarePants** [comment, marked as answer]
Already exists in our V2 MyGet development builds. Set `BmpBitsPerPixel.Pixel1` during encode. 

Note: ImageSharp comes with a [Commercial License](https://sixlabors.com/pricing/) for professional projects.
